dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -215,6 +215,10 @@ def authenticate
           render 'lti/v1/upgrade_account' and return
         end
 
+        if decoded_jwt[Policies::Lti::MessageType::CLAIM] == Policies::Lti::MessageType::DEEP_LINKING_REQUEST
+          redirect_to lti_v1_deep_linking_index_path deep_linking_settings: decoded_jwt[Policies::Lti::DEEP_LINKING_SETTINGS_CLAIM] and return
```

Comment:
Please use parentheses to make it a bit easier to see which method the optional arguments belong to:
```suggestion
          redirect_to lti_v1_deep_linking_index_path(deep_linking_settings: decoded_jwt[Policies::Lti::DEEP_LINKING_SETTINGS_CLAIM]) and return
```
```suggestion
          deep_linking_settings = decoded_jwt[Policies::Lti::DEEP_LINKING_SETTINGS_CLAIM]
          redirect_to lti_v1_deep_linking_index_path(deep_linking_settings:) and return
```